plugin/backend_test.go

```diff
@@ -60,12 +60,13 @@ func WriteConfig(t *testing.T) {
 		Path:      configPath,
 		Storage:   testStorage,
 		Data: map[string]interface{}{
-			"binddn":      "tester",
-			"password":    "pa$$w0rd",
-			"url":         "ldap://138.91.247.105",
-			"certificate": validCertificate,
-			"userdn":      "dc=example,dc=com",
-			"formatter":   "mycustom{{PASSWORD}}",
+			"binddn":                       "tester",
+			"password":                     "pa$$w0rd",
+			"url":                          "ldap://138.91.247.105",
+			"certificate":                  validCertificate,
+			"userdn":                       "dc=example,dc=com",
+			"formatter":                    "mycustom{{PASSWORD}}",
+			"out_of_band_rotation_seconds": 5,
```

Comment:
I don't think the name should have `_seconds`, since I can just as easily pass in "1m" or "2h" and it will be parsed correctly.

And given that, maybe `last_rotation_tolerance` would make more sense. I think "out_of_band" implies more going on that what is more likely the case of some lag propagating the change.

Reply:
Yes, I like that. I was having trouble coming up with a name that was both clear and concise. I do think tolerance is a much more concise word that gets the concept across. I'll update it. Thanks!